### DISCUSSION TITLE
How to add validations for group of two text fields i need either this text field to be fill or another text field

### DISCUSSION THREAD

**Discussion by karthiktrp102** [react-hook-form/react-hook-form#5457] [General]
How to add validations for group of two text fields i need either this text field to be fill or another text field

**Comment by bluebill1049** [comment]
https://react-hook-form.com/api/useform/register

use `validate` function.

**Reply by karthiktrp102** [reply]
can you explain a bit more how to exactly validate the two text fields inside the same controller

**Reply by thanh-nguyen-95** [reply]
https://github.com/react-hook-form/react-hook-form/discussions/5491#discussioncomment-826529

Answered in this discussion. Please be patient and wait for a response instead of creating another discussion. If you have any further problem, please discuss it in the other topic.
